sdk/inc/azure/core/az_mqtt5_rpc_server_codec.h

```diff
@@ -27,21 +27,21 @@
 typedef struct
 {
   /**
-   * @brief Timeout in seconds for subscribing acknowledgement.
-   */
-  int32_t subscribe_timeout_in_seconds;
-
-  /**
-   * @brief An #az_span containing the service group id to use for the subscription topic, can be
-   * AZ_SPAN_EMPTY if not using a shared subscription topic.
+   * @brief The service group id to use for the subscription topic, specifying this allows the
```

Comment:
Can you keep the part of the comment where this can be AZ_SPAN_EMPTY if you don't want to specify it?